src/pages/[platform]/start/project-setup/prerequisites/index.mdx

```diff
@@ -19,6 +19,12 @@ export function getStaticProps(context) {
   };
 }
 
+<Callout info>
+
+Project setup guides are available for the [previous version](/[platform]/prev/start/project-setup/prerequisites/). However, it is highly recommended that you use the latest version to set up new projects following this guide.
```

Comment:
Change second sentence to "However, we highly recommend using the latest version and following this guide when setting up new projects."